cobbler/utils/filesystem_helpers.py

```diff
@@ -244,6 +245,24 @@ def copyremotefile(src: str, dst1: str, api: Optional["CobblerAPI"] = None) -> N
         ) from error
 
 
+def copyfileimage(src: str, image_location: str, dst: str) -> None:
+    """
+    Copy a file from source to the destination in the image.
+
+    :param src: The source file.
+    :param image_location: The location of the image.
+    :param dst: The destination for the file.
+    """
+    cmd = ["mcopy", "-n, "-i", image_location, src, dst]
```

Comment:
Yay! :smile: It seems you have a syntax error here:

```suggestion
    cmd = ["mcopy", "-n", "-i", image_location, src, dst]
```